dtfs-central-api/src/helpers/get-keying-sheet-calculation-facility-values.ts

```diff
@@ -4,34 +4,37 @@ import { TfmFacilitiesRepo } from '../repositories/tfm-facilities-repo';
 import { NotFoundError } from '../errors';
 import { convertTimestampToDate } from './convert-timestamp-to-date';
 import { getEffectiveCoverEndDateAmendment } from './amendments/get-effective-cover-end-date-amendment';
+import { KeyingSheetCalculationFacilityValues } from '../types/tfm/tfm-facility';
 
 /**
- * Gets the latest values for the TFM facility with the supplied facility id
+ * Gets TFM facility values with the supplied facility id for keying sheet calculations
+ * if a report period is supplied, the values returned will take into account any amendments that came into effect before the end of the report period passed in.
+ * else will return the values as they were at facility creation ignoring all amendments
```

Comment:
Since we are not taking into account value amendments like I had requested this needs updating as is not true in that case